[0.017s] Invoking command in '/home/alip/clover_ws/src/build/usb_camera_driver': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/foxy /usr/bin/cmake /home/alip/clover_ws/src/ros2_usb_camera -DAMENT_CMAKE_SYMLINK_INSTALL=1 -DCMAKE_INSTALL_PREFIX=/home/alip/clover_ws/src/install/usb_camera_driver
[0.057s] -- The C compiler identification is GNU 9.4.0
[0.096s] -- The CXX compiler identification is GNU 9.4.0
[0.101s] -- Check for working C compiler: /usr/bin/cc
[0.149s] -- Check for working C compiler: /usr/bin/cc -- works
[0.150s] -- Detecting C compiler ABI info
[0.201s] -- Detecting C compiler ABI info - done
[0.219s] -- Detecting C compile features
[0.219s] -- Detecting C compile features - done
[0.222s] -- Check for working CXX compiler: /usr/bin/c++
[0.285s] -- Check for working CXX compiler: /usr/bin/c++ -- works
[0.286s] -- Detecting CXX compiler ABI info
[0.343s] -- Detecting CXX compiler ABI info - done
[0.352s] -- Detecting CXX compile features
[0.353s] -- Detecting CXX compile features - done
[0.355s] -- Found ament_cmake: 0.9.11 (/opt/ros/foxy/share/ament_cmake/cmake)
[0.373s] -- Found PythonInterp: /usr/bin/python3 (found suitable version "3.8.10", minimum required is "3") 
[0.374s] -- Using PYTHON_EXECUTABLE: /usr/bin/python3
[0.521s] -- Override CMake install command with custom implementation using symlinks instead of copying resources
[0.596s] -- Found rclcpp: 2.4.2 (/opt/ros/foxy/share/rclcpp/cmake)
[0.828s] Invoked command in '/home/alip/clover_ws/src/build/usb_camera_driver' returned '-2': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/foxy /usr/bin/cmake /home/alip/clover_ws/src/ros2_usb_camera -DAMENT_CMAKE_SYMLINK_INSTALL=1 -DCMAKE_INSTALL_PREFIX=/home/alip/clover_ws/src/install/usb_camera_driver
[0.901s] Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/colcon_parallel_executor/executor/parallel.py", line 67, in execute
    loop.run_until_complete(future)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 603, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.8/selectors.py", line 468, in select
    fd_event_list = self._selector.poll(timeout, max_ev)
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__
    rc = await self.task(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__
    return await task_method(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_cmake/build.py", line 59, in build
    rc = await extension.build(
  File "/usr/lib/python3/dist-packages/colcon_cmake/task/cmake/build.py", line 87, in build
    rc = await self._reconfigure(args, env)
  File "/usr/lib/python3/dist-packages/colcon_cmake/task/cmake/build.py", line 182, in _reconfigure
    buildfile = get_buildfile(cmake_cache)
  File "/usr/lib/python3/dist-packages/colcon_cmake/task/cmake/__init__.py", line 112, in get_buildfile
    if 'Ninja' in generator:
TypeError: argument of type 'NoneType' is not iterable
